### DISCUSSION TITLE
Can I use this without grub, with another bootloader?

### DISCUSSION THREAD

**Discussion by is0ly** [prasanthrangan/hyprdots#721] [Q&A] [answered]
The actual question is in the title. Can I use this without grub, with another bootloader? opencore, rEFInd? 
I'm sorry if this is a dumb question.

**Comment by kRHYME7** [comment, marked as answer]
Yes, it only modifies grub and systemdboot,
But if you're using Nvidia drivers refer first to https://wiki.hyprland.org to the Nvidia section.  (

**Reply by is0ly** [reply]
I have a red card)

**Reply by kRHYME7** [reply]
Can you please elaborate more?

**Reply by is0ly** [reply]
amd card,
